integration-tests/tests/src/tests/realm-constructor.ts

```diff
@@ -204,19 +211,35 @@ describe("Realm#constructor", () => {
       }).throws(message);
     }
 
-    it("fails when asking for a list of lists", () => {
+    // Old test: The new schema parser no longer allows combining shorthand and object notation.
+    it.skip("fails when asking for a list of lists", () => {
```

Comment:
I added `.skip` to some of the tests that were using schemas that are now invalid. A new test was added below. Not sure if we want to keep the `.skip` tests for reference or just delete them?

Reply:
They can be deleted, just as we'll delete code instead of commenting it out. We can always use Git if we want to time-travel back and look at this.